scripts/ci-subtask.sh

```diff
@@ -3,34 +3,51 @@
 # ./ci-subtask.sh <TOTAL_TASK_N> <TASK_INDEX>
 
 ROOT_PATH_COV=$(pwd)/covprofile
+ROOT_PATH_JUNITFILE=$(pwd)/junitfile
 # Currently, we only have 3 integration tests, so we can hardcode the task index.
 integrations_dir=$(pwd)/tests/integrations
 
 case $1 in
     1)
-        # unit tests ignore `tests`
-        ./bin/pd-ut run --race --ignore tests --coverprofile $ROOT_PATH_COV || exit 1
+        # unit tests ignore `tests`, `server`, `schedule`,`utils` and `encryption`
+        ./bin/pd-ut run --ignore tests,server,pkg/schedule,pkg/utils,pkg/encryption --race --coverprofile $ROOT_PATH_COV --junitfile $ROOT_PATH_JUNITFILE || exit 1
```

Comment:
What if these packages are deleted or renamed?

Reply:
It may be necessary to manually change the code,when noticing abnormal `test time`/`test codecov`

Reply:
- codev can reveal missing tests
- Redundant tests can be reacted to at test time